playbook/app/pb_kits/playbook/pb_circle_icon_button/_circle_icon_button.jsx

```diff
@@ -31,6 +32,7 @@ type CircleIconButtonProps = {
 const CircleIconButton = (props: CircleIconButtonProps) => {
   const {
     aria = {},
+    buttonData = {},
```

Comment:
Doesn't break anything already implemented. Now send `buttonData` to add data to the button directly.